src/Microsoft.Health.Fhir.Anonymizer.Shared.Core/AnonymizerConstants.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Fhir.Anonymizer.Shared.Core
+{
+    public static class AnonymizerConstants
+    {
+        public const string FhirLibraryVersion = "3.4.0";
```

Comment:
Are we going to check the version number in de-id export?

Reply:
yes, we are going to check this version to make sure they are same.

Reply:
Is there any method to get this value in a more elegant way instead of hard coding it?